メールが重複した際にエラーメッセージが表示されるように修正

ryotaro-rks opened the pull request.

ryotaro-rks committed:
メールが重複した際にエラーメッセージが表示されるように修正

igamasayuki commented on src/main/java/jp/co/sample/emp_management/controller/AdministratorController.java:
```diff
@@ -72,16 +72,20 @@ public String toInsert() {
+		} else {
+			model.addAttribute("duplicateMessage", "既に登録されているメールアドレスです");
+			return toInsert();
+		}
```
　(1-3)中級 Eメール重複チェック
model.addAttribute("duplicateMessage", "既に登録されているメールアドレスです");

ではなく、今回　resultにエラーメッセージを追加して、

viewでは

<label th:errors="*{mailAddress}" class="error-messages">
											メールアドレスを入力してください
</label>

で共通して必須入力のエラーメッセージと一緒にエラーメッセージを出すことができます。
BindingResultのAPIマニュアルを見ながら探してみてください。

ryotaro-rks replied:
レビューありがとうございます。修正いたします

igamasayuki reviewed (changes requested):
修正依頼を出しましたので修正後Slackにてご連絡をお願いします。

ryotaro-rks committed:
Merge branch 'develop' into fix/check-mail-duplicate

ryotaro-rks committed:
エラーメッセージをBindingResultに入れるように修正

igamasayuki committed:
エラーメッセージを共通で出力できるようにしたため、別途設置していたメッセージ出力用のタグを削除

insert.htmlの59行目の以下が不要になるため削除してコミットしておきました。

<p th:text="${duplicateMessage}"></p>

igamasayuki reviewed (approved):
問題ありませんので承認します。

insert.htmlの59行目の以下が不要になるため削除してコミットしておきました。
<p th:text="${duplicateMessage}"></p>

履歴にて変更箇所をご確認ください。
